XenAdmin/Wizards/PatchingWizard/PatchingWizard.cs

```diff
@@ -296,8 +284,11 @@ protected override void OnCancel()
         {
             base.OnCancel();
 
-            var subActions = BuildSubActions(PatchingWizard_PrecheckPage.GetUnwindChangesActions,
-                GetRemovePatchActions, GetRemoveVdiActions, GetCleanUpPoolUpdateActions);
+            var subActions = new List<AsyncAction>();
+            subActions.AddRange(GetUnwindChangesActions(PatchingWizard_PrecheckPage.PrecheckProblemsActuallyResolved));

```

Comment:
AddRange fails with ArgumentNullException if any of these functions (GetUnwindChangesActions, GetRemovePatchActions, etc.) returns null

Reply:
oops! 😳